src/databricks/labs/lsql/lakeview/model.py

```diff
@@ -1681,6 +1681,7 @@ def as_dict(self) -> Json:
         body: Json = {
             "version": 1,
             "widgetType": "table",
+            "invisibleColumns": [v.as_dict() for v in self.invisible_columns],
```

Comment:
@nfx : fyi, `invisibleColumns` always needs to be present. I made the changes to do so. In hindsight maybe not unexpected as it is a required field. The code generation could maybe take this into account, always put the required fields in the `as_dict` body